test/integration_tests/long/test_model_forecasting.py

```diff
@@ -143,7 +156,13 @@ def test_forecast_with_column_rename(self):
         self.assertEqual(len(result), 24)
         self.assertEqual(
             result.columns,
-            ["homeforecast.type", "homeforecast.saledate", "homeforecast.ma"],
+            [
+                "homeforecast.type",
+                "homeforecast.saledate",
+                "homeforecast.ma",
+                "homeforecast.ma-lo",
+                "homeforecast.ma-hi",
+            ],
```

Comment:
To save time, we can keep one testcases for neural forecasting and one for statsforecast. Does the feedback add a lot of execution overhead? We did not timeout for a 30 minutes test before.

Reply:
Hmm, this a little weird. The feedback system is turned off by default for `neuralforecast`.

Reply:
There are two statsforecast tests in the test_model_forecasting. The feedback system is running for those two tests. That takes time?

Reply:
Not really. Crossvalidation for statsforecast is usually quick. I'll take a look.

Reply:
This is the exact time out error. I'll try to figure out the cause:

```
FAILED test/integration_tests/long/test_model_forecasting.py::ModelTrainTests::test_forecast - evadb.executor.executor_utils.ExecutorError: Timed out initializing process group in store based barrier on rank: 0, for key: store_based_barrier...
```

Perhaps it's something to do with pytorch?
https://discuss.pytorch.org/t/timed-out-initializing-process-group-in-store-based-barrier/119803/20

Reply:
I do not think that pytorch is used in statsforecasting. Is the cross validation working locally for you?